file_system/models.py

```diff
@@ -2,7 +2,7 @@
 import uuid
 from enum import IntEnum
 from django.db import models
-from django.contrib.postgres.fields import JSONField
+from django.contrib.postgres.fields import JSONField, ArrayField
```

Comment:
did you plan to use `ArrayField` somewhere in here?

Reply:
I was planing and then forgot to remove import when I decided for different implementation. Will remove it. Thanks